docs/source/_src_about/history.rst

```diff
@@ -0,0 +1,55 @@
+
+**First steps**
+
+The scope of the software has evolved since then. Although it was initially conceptualized to primarily support trajectory optimization problems, the codebase evolved more towards supporting numerical propagation problems, both in an optimization loop, and outside of it. Since starting the software, it has gone through a number of significant evolutionary stages. Several years into the project, a thorough rewrite of much of the internal workings was undertaken, based on our own experience and input from several external coding experts. Around 2012, it was decided to use Tudat as the core tool for WP3 (Laser Ranging) of the European ESPaCE project. In the context of this project, functionality was developed to support batch least squares state estimation, first only from laser ranging data but later on from more general data types. This functionality was applied to a number of real and simulated test cases, and led to a number or publications. After this project ended, Tudat continued to be used in this context, as the main tool for simulating the contribution of the PRIDE experiment on the JUICE mission, an ongoing collaboration between TU Delft, JIVE, and others.
+
+**Improving accessibility**
+
+In 2016, another substantial addition was made to Tudat: the 'simulation setup' layer. Before, a user would manually have to link all their objects together manually, which ended being very time-consuming and required significant C++ expertise on the side of the user. Since much of the user base was then TU Delft M.Sc. students, this led to significant issues, as the expected/required skills of students and their background did not match. The simulation setup layer (which is now the ``numerical_simulation.environment_setup/propagation_setup/estimation_setup`` submodule) made the use of Tudat much easier, by no longer requiring as much C++ expertise. Still, the required coding expertise to use Tudat was daunting. Around the same time, Tudat was migrated from a privately hosted website, and moved onto Github. Also, Tudat was incorporated into the Space Exploration curriculum of TU Delft further, first in a course named 'Space Project', now renamed to 'Numerical Astrodynamics'. Initially, this course was meant primarilly to give students academic credit for the time they put into getting familiar with Tudat. Now, it is a full-fledged course which focusses on setting up and interpreting astrodynamics problems, where Tudat is 'merely' the tool that happens to be used. A year later, this course was supplemented by the 'Propagation and Optimization in Astrodynamics' course, where the focus is on selecting suitable models for a numerical problem, where we use both Tudat and Pagmo/Pygmo.
```

Comment:
All of these new edits are great, especially the history page that makes this whole software package a little more human 🧑‍🚀
The only edit I would make would be to add links when talking about the [Numerical Astrodynamics](https://studiegids.tudelft.nl/a101_displayCourse.do?course_id=60207) and [Propagation and Optimization in Astrodynamics](https://studiegids.tudelft.nl/a101_displayCourse.do?course_id=60206) courses, and [Pygmo](https://esa.github.io/pygmo2/). This is for two reasons: Google loves if we reference different websites, and maybe some intrigued users may need just this little push to get interested in following these courses.